tests/test_plugins/test_plugin/server.py

```diff
@@ -22,31 +22,33 @@
 from girder.api.describe import Description, describeRoute
 from girder.api.rest import boundHandler, rawResponse, Resource, setResponseHeader
 from girder.api.v1.collection import Collection
+from girder.constants import TokenScope
 from girder.utility.server import staticFile
 
 
-@access.public
+@access.user(scope=TokenScope.ANONYMOUS_SESSION)
```

Comment:
The point of changing these tests is to present a more challenging test case, which fails before the subsequent commits in this PR are added.

Previously, when applied with the argument-calling syntax, `boundHandler` was not exposing some special properties (e.g. `__name__`) on the new function. When the `access.user` decorator subsequently re-wraps the `boundHandler`-wrapped function, the  `six.wraps` utility cannot find these special properties and fails.

The `access.public` decorator only attaches some additional properties to the target function, rather than re-wrapping it, so it does not trigger this sort of failure.

Finally, the addition of `TokenScope.ANONYMOUS_SESSION` is to ensure that the logic inside the applied decorators is actually being called.